builder/amazon/common/run_config.go

```diff
@@ -96,12 +96,27 @@ type RunConfig struct {
 	// `true`, will cause a timeout.
 	// Example of a valid shutdown command:
 	//
+	// <Tabs>
+	// <Tab heading="JSON">
+	//
 	// ```json
-	// {
+	// "provisioners: [{
 	//   "type": "windows-shell",
 	//   "inline": ["\"c:\\Program Files\\Amazon\\Ec2ConfigService\\ec2config.exe\" -sysprep"]
+	// }]
+	// ```
+	//
+	// </Tab>
+	// <Tab heading="HCL2">
+	//
+	// ```hcl
+	// provisioner "windows-shell"{
+	//   inline = ["\"c:\\Program Files\\Amazon\\Ec2ConfigService\\ec2config.exe\" -sysprep"]
```

Comment:
I think this should work with HCL2:

```suggestion
	//   inline = ["c:/Program Files/Amazon/Ec2ConfigService/ec2config.exe -sysprep"]
```